src/odemis/driver/smarpod.py

```diff
@@ -0,0 +1,873 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 17 April 2019
+
+@author: Anders Muskens
+
+Copyright © 2012-2019 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+This driver supports SmarAct and SmarPod actuators, which are accessed via a C DLL library
+provided by SmarAct. This must be installed on the system for this actuator to run. Please
+refer to the SmarAct readme for Linux installation instructions.
+'''
+from __future__ import division
+from concurrent.futures import CancelledError, TimeoutError
+
+import os
+import logging
+import time
+import math
+from ctypes import *
+import copy
+import threading
+
+from odemis import model
+from odemis.util import driver
+from odemis.model import HwError, CancellableFuture, CancellableThreadPoolExecutor, isasync
+from operator import pos
+
+
+def add_coord(pos1, pos2):
+    """
+    Adds two coordinate dictionaries together and returns a new coordinate dictionary.
+
+    NOTE: pos1 and pos2 MUST contain the same axes values
+
+    pos1: dict (axis name str) -> (float)
+    pos2: dict (axis name str) -> (float)
```

Comment:
Please update the docstrings.

Reply:
Please, state in the docstring that all pos2 keys must be present in pos1, and that the ret keys are the same as in pos2.